tests/pytorch/test_pytorch_autolog.py

```diff
@@ -1,392 +1,28 @@
 from mlflow.tracking.client import MlflowClient
 
 NUM_EPOCHS = 20
 
-pytestmark = pytest.mark.large
 
+def test_pytorch_autolog_logs_expected_data(tmpdir):
+    from torch.utils.tensorboard import SummaryWriter
+    mlflow.pytorch.autolog_vanilla()
+    writer = SummaryWriter(str(tmpdir))
 
-@pytest.fixture
-def pytorch_model():
-    mlflow.pytorch.autolog()
-    model = IrisClassification()
-    dm = IrisDataModule()
-    dm.setup(stage="fit")
-    trainer = pl.Trainer(max_epochs=NUM_EPOCHS)
-    trainer.fit(model, dm)
-    client = mlflow.tracking.MlflowClient()
-    run = client.get_run(client.list_run_infos(experiment_id="0")[0].run_id)
-    return trainer, run
-
-
-@pytest.fixture
-def pytorch_model_without_validation():
-    mlflow.pytorch.autolog()
-    model = IrisClassificationWithoutValidation()
-    dm = IrisDataModuleWithoutValidation()
-    dm.setup(stage="fit")
-    trainer = pl.Trainer(max_epochs=NUM_EPOCHS)
-    trainer.fit(model, dm)
-    client = mlflow.tracking.MlflowClient()
-    run = client.get_run(client.list_run_infos(experiment_id="0")[0].run_id)
-    return trainer, run
-
-
-@pytest.fixture(params=[(1, 1), (1, 10), (2, 1)])
-def pytorch_model_with_steps_logged(request):
-    log_every_n_epoch, log_every_n_step = request.param
-    mlflow.pytorch.autolog(log_every_n_epoch=log_every_n_epoch, log_every_n_step=log_every_n_step)
-    model = IrisClassification()
-    dm = IrisDataModule()
-    dm.setup(stage="fit")
-    trainer = pl.Trainer(max_epochs=NUM_EPOCHS)
-    trainer.fit(model, dm)
-    client = mlflow.tracking.MlflowClient()
-    run = client.get_run(client.list_run_infos(experiment_id="0")[0].run_id)
-    return trainer, run, log_every_n_epoch, log_every_n_step
-
-
-@pytest.mark.parametrize("log_models", [True, False])
-def test_pytorch_autolog_log_models_configuration(log_models):
-    mlflow.pytorch.autolog(log_models=log_models)
-    model = IrisClassification()
-    dm = IrisDataModule()
-    dm.setup(stage="fit")
-    trainer = pl.Trainer(max_epochs=NUM_EPOCHS)
-    trainer.fit(model, dm)
-    client = mlflow.tracking.MlflowClient()
-    run = client.get_run(client.list_run_infos(experiment_id="0")[0].run_id)
-    run_id = run.info.run_id
-    client = mlflow.tracking.MlflowClient()
-    artifacts = [f.path for f in client.list_artifacts(run_id)]
-    assert ("model" in artifacts) == log_models
-
-
-def test_pytorch_autolog_logs_default_params(pytorch_model):
-    _, run = pytorch_model
-    data = run.data
-    assert "lr" in data.params
-    assert "eps" in data.params
-    assert "optimizer_name" in data.params
-    assert "weight_decay" in data.params
-    assert "betas" in data.params
-
-
-def test_pytorch_autolog_logs_expected_data(pytorch_model):
-    _, run = pytorch_model
-    data = run.data
+    with mlflow.start_run():
+        for i in range(NUM_EPOCHS):
+            writer.add_scalar('loss', 42.0, global_step=i)
+        writer.close()
```

Comment:
Could you add similar test coverage that `pytorch_lightning` has to verify that metrics and params are logged properly with this autologging 'flavor'?